cmd/apprepository-controller/main_test.go

```diff
@@ -0,0 +1,153 @@
+package main
```

Comment:
As far as I can tell, this is just testing that the flags behave as expected (which is not needed if we use a different package).